src/backend/prisma/migrations/20230702055640_init/migration.sql

```diff
@@ -0,0 +1,30 @@
+-- CreateTable
+CREATE TABLE "project" (
+    "id" TEXT NOT NULL,
+    "name" VARCHAR(30) NOT NULL,
+    "description" VARCHAR(100) NOT NULL,
```

Comment:
Sure you want to limit the description to 100? Doesn't feel like it's something that needs limitation.

Reply:
I think it's a good practice to always limit the length of your db inputs (for security reasons etc.)

worst case scenario I can always do another migration with a larger length.

I think its better to start strong and let go than start loose cause its almost impossible to change the data later on